### PR TITLE
pokecli.py wait_time not defined

### PR DESCRIPTION
**Pull request by Gobberwart**
## Short Description:

NoPlayerPositionSetException tries to sleep for "wait_time" but "wait_time" isn't defined. Causes bot crash.

## Fixes/Resolves/Closes (please use correct syntax):
- #5280

### CODE REVIEW

**Comment by mjmadsen** [comment]
Been working to extend this and others. Could you move it up a level or two? A few of the below exceptions use it

**Comment by Gobberwart** [comment]
@mjmadsen not sure what you mean? wait_time defined above all the exceptions?

**Comment by mjmadsen** [comment]
Yeah, it's declared in an except, but used in a few excepts out of scope. One of the many problems in this file.

**Comment by Gobberwart** [comment]
Yeah I'm just doing bandaids at the moment. How's that latest commit look?

File: pokecli.py [hunk 1]
```diff
@@ -213,6 +213,7 @@ def get_commit_hash():
                 )
                 time.sleep(36000)
             except NoPlayerPositionSetException:
+                wait_time = config.reconnecting_timeout * 60
```
Comment (mjmadsen) [inline pokecli.py]: We need to move the scope of this. We use it further down the line with different excepts.

**Comment by Gobberwart** [comment]
@mjmadsen Fair enough. I'm just doing bandaids right now, but agree no need to define it multiple times. How's the latest commit look?

**Comment by solderzzc** [comment]
@mjmadsen  how do you think about our new gitter channel ?
https://github.com/PokemonGoF/PokemonGo-Bot#development-channel

**Comment by Gobberwart** [comment]
Oi @solderzzc no thread-jacking! :)

**Comment by solderzzc** [comment]
Another breaking :)

**Comment by mjmadsen** [comment]
@Gobberwart I get it, but I realized we were defining the sleep timer in a  place that can't access the declaration. I've been staring at this file all day. I am working on a PR that breaks the functions up. But one of the big things I noticed is that that variable is defined in an exception and used in other exceptions - thus needs to be defined before all of all exceptions that use it.

**Comment by Gobberwart** [comment]
@mjmadsen I completely understand what you're saying. I've defined wait_timeout before the try block starts. Does that work for you?

**Comment by mjmadsen** [comment]
My notes from looking at this file today:
We need to break non essential config stuff to a config handler file (and clean up our general config checks - move checks closer to load).
Break our try statements into smaller chunks - we have absurd try blocks.
Adapt python idioms (we don't apply the best methods on many file handling bits). 
Use exceptions properly to give us the correct traceback. 
Break up certain functions to allow better try/except blocks

**Comment by Gobberwart** [comment]
@mjmadsen Agree 100%. This is just a bandaid fix though. OK to merge?

**Comment by mjmadsen** [comment]
:+1:

**Comment by mjmadsen** [comment]
I'll merge after CI passes (though I know it will). Sorry to be dismal, I've spent too much time looking at pokecli.py being frustrated today. I have many wants from looking at that file.

**Comment by mjmadsen** [comment]
Showing errors:
FILE: /home/travis/build/PokemonGoF/PokemonGo-Bot/pokecli.py
FAILED pylint inspection: E:173, 0: unindent does not match any outer indentation level (syntax-error)

**Comment by Gobberwart** [comment]
@mjmadsen Yep, but that's nothing to do with this PR. Seems config examples are busted.

**Comment by Gobberwart** [comment]
@mjmadsen OK fixed

**Comment by mjmadsen** [comment]
:+1:

**Pull request merged**